src/scss/application.scss

```diff
@@ -16,6 +16,10 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+a {
+  width: 100%;
+}
```

Comment:
are u sure that you want that all anchor tags uses 100% of the available space?
this is wrong, what if you have a text then a link, then a text, then a link?